ktor-version-catalog/build/version-catalog/libs.versions.toml

```diff
@@ -0,0 +1,125 @@
+server-jetty-jakarta = {group = "io.ktor", name = "ktor-server-jetty-jakarta", version.ref = "ktor" }
+server-jte = {group = "io.ktor", name = "ktor-server-jte", version.ref = "ktor" }
+server-methodOverride = {group = "io.ktor", name = "ktor-server-method-override", version.ref = "ktor" }
+server-metrics = {group = "io.ktor", name = "ktor-server-metrics", version.ref = "ktor" }
+server-metrics-micrometer = {group = "io.ktor", name = "ktor-server-metrics-micrometer", version.ref = "ktor" }
+server-mustache = {group = "io.ktor", name = "ktor-server-mustache", version.ref = "ktor" }
+server-netty = {group = "io.ktor", name = "ktor-server-netty", version.ref = "ktor" }
+server-openapi = {group = "io.ktor", name = "ktor-server-openapi", version.ref = "ktor" }
+server-partialContent = {group = "io.ktor", name = "ktor-server-partial-content", version.ref = "ktor" }
+server-pebble = {group = "io.ktor", name = "ktor-server-pebble", version.ref = "ktor" }
+server-rateLimit = {group = "io.ktor", name = "ktor-server-rate-limit", version.ref = "ktor" }
+server-requestValidation = {group = "io.ktor", name = "ktor-server-request-validation", version.ref = "ktor" }
+server-resources = {group = "io.ktor", name = "ktor-server-resources", version.ref = "ktor" }
+server-servlet = {group = "io.ktor", name = "ktor-server-servlet", version.ref = "ktor" }
+server-servlet-jakarta = {group = "io.ktor", name = "ktor-server-servlet-jakarta", version.ref = "ktor" }
+server-sessions = {group = "io.ktor", name = "ktor-server-sessions", version.ref = "ktor" }
+server-sse = {group = "io.ktor", name = "ktor-server-sse", version.ref = "ktor" }
+server-statusPages = {group = "io.ktor", name = "ktor-server-status-pages", version.ref = "ktor" }
+server-swagger = {group = "io.ktor", name = "ktor-server-swagger", version.ref = "ktor" }
+server-testHost = {group = "io.ktor", name = "ktor-server-test-host", version.ref = "ktor" }
+server-thymeleaf = {group = "io.ktor", name = "ktor-server-thymeleaf", version.ref = "ktor" }
+server-tomcat = {group = "io.ktor", name = "ktor-server-tomcat", version.ref = "ktor" }
+server-tomcat-jakarta = {group = "io.ktor", name = "ktor-server-tomcat-jakarta", version.ref = "ktor" }
+server-velocity = {group = "io.ktor", name = "ktor-server-velocity", version.ref = "ktor" }
+server-webjars = {group = "io.ktor", name = "ktor-server-webjars", version.ref = "ktor" }
+server-websockets = {group = "io.ktor", name = "ktor-server-websockets", version.ref = "ktor" }
+sse = {group = "io.ktor", name = "ktor-sse", version.ref = "ktor" }
+testDispatcher = {group = "io.ktor", name = "ktor-test-dispatcher", version.ref = "ktor" }
+utils = {group = "io.ktor", name = "ktor-utils", version.ref = "ktor" }
+websocket-serialization = {group = "io.ktor", name = "ktor-websocket-serialization", version.ref = "ktor" }
+websockets = {group = "io.ktor", name = "ktor-websockets", version.ref = "ktor" }
```

Comment:
We can also create an alias `websockets-serialization` pointing to `ktor-websocket-serialization`. (Probably this project was named without `s` at the end by a mistake)